eve/main.yml

```diff
@@ -90,19 +90,19 @@ stages:
       flavor: m1.medium
       path: eve/workers/openstack-single-node
     steps:
-      - ShellCommand:
+      - ShellCommand: &retrieve_iso
```

Comment:
Minor: since we're starting to reuse these, we could have a YAML section dedicated to reused snippets - see RING `main.yaml` for an example of that.

Reply:
Will keep that in mind for next steps, indeed it could be cool to merge some common steps!